### ISSUE TITLE
rustdesk is online on all machines, but message says it's offline

### ISSUE THREAD

**Issue by Richirat** [rustdesk/rustdesk#4845]
### Bug Description

Everything is up and running as usual, Rustdesk is online on all machines, though I cannot connect to my favorites and a message says: Remote desktop is offline. Tried to reinstall Rustdesk but didn't change anything. 

### How to Reproduce

I don't know exactly. It always worked fine, but today.

### Expected Behavior

I expected to connect to my saved PCs fast and cleanly, as usual. error message appeared

### Operating system(s) on local side and remote side

windows 11 local, windows 7 remote

### RustDesk Version(s) on local side and remote side

1.1.9 installed on both PCs

### Screenshots

![Rustdesk_bug](https://github.com/rustdesk/rustdesk/assets/83584944/38b339c3-0dec-4272-8bb5-8dd7af925c80)

### Additional Context

 PCs reconnected after many failed attempts and offline messages.

**Comment by General1337** [comment]
Also having this issue, seems like it's from RustDesk server end.

**Comment by jmim25** [comment]
me too

**Comment by Cyborgscode** [comment]
since 1.5h a new major release is out. the server system **-> may have been <-** upgraded to work with that version alone. Pls test it with 1.2.0

**Comment by jkhsjdhjs** [comment]
Same issue with 1.2.0 on Arch Linux.

**Comment by pixelstuff** [comment]
Seeing the same issue. Can't connect to any devices, yet on those devices RustDesk shows Ready.  I have some 1.1.9 clients and some new 1.2.0 clients. All the same.

**Comment by dinger1986** [comment]
any ideas what server you are being connected with?

**Comment by jkhsjdhjs** [comment]
Yes, it's `155.138.247.159` for me. I just set up my own ID and relay server and now it's working.
